app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextActions.java

```diff
@@ -191,7 +191,11 @@ private boolean onClickImpl(final View view) {
                     });
                     return true;
                 }
-                case R.string.tmaid_common_indent:
+                case R.string.tmaid_common_indent: {
+                    if (_appSettings.isMarkdownAutoUpdateList()) {
+                        runPrefixReplaceAction(PREFIX_ORDERED_LIST, "$0", "$11$4 ");
```

Comment:
Not strictly related to insert list. This makes sure ordered list numbering is restarted on indent.

Reply:
this looks very cryptic, very sure don't know what it this line this does later on. I don't understand fully either even now. Can you please add a comment there?